### PR TITLE
basic deployment plugin mocked by SinonJs

### PR DESCRIPTION
**Pull request by eahefnawy**

### CODE REVIEW

**Comment by flomotlik** [comment]
@eahefnawy please resolve the conflicts, then we can merge

File: lib/plugins/deploy/tests/deploy.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+'use strict';
```
Comment (flomotlik) [inline lib/plugins/deploy/tests/deploy.js]: Can you add this to all.js right away so the tests are guaranteed to run

Comment (eahefnawy) [inline lib/plugins/deploy/tests/deploy.js]: done

**Comment by eahefnawy** [comment]
thank you coveralls! 💃

**Pull request merged**